codersdk/deployment.go

```diff
@@ -1862,7 +1865,9 @@ const (
 // users to opt-in to via --experimental='*'.
 // Experiments that are not ready for consumption by all users should
 // not be included here and will be essentially hidden.
-var ExperimentsAll = Experiments{}
+var ExperimentsAll = Experiments{
+	ExperimentTemplateInsightsPage,
```

Comment:
Is it intended to release as experimental?

Reply:
Yes, I talked with @bpmct about this